src/learn/qml/MuseScore/Learn/internal/ClassesPage.qml

```diff
@@ -27,8 +27,10 @@ import QtGraphicalEffects 1.0
 import MuseScore.Ui 1.0
 import MuseScore.UiComponents 1.0
 
-FocusScope {
+StyledFlickable {
     id: root
+    contentWidth: parent.width
+    contentHeight: authorInfo.height + sideMargin
```

Comment:
Our style is usually to put new property declarations at the top, and property assignments below that. That would mean moving these two properties below `property int sideMargin: …`.